packages/web-extension/package.json

```diff
@@ -60,6 +59,7 @@
   },
   "dependencies": {
     "@cardano-sdk/core": " 0.2.0",
+    "@cardano-sdk/cip30": " 0.2.0",
```

Comment:
I can see why this is being done to untangle the deps, but this is really the inverse of what we want right? Maybe log a spike to review how we can achieve the goal of a more generic `web-extension` package that `cip30` depends on. Maybe we can consider a pattern of `{PACKAGE_NAME}-dev` for smaller aligned scoping.

Reply:
Depends how you look at it. I think current dependency tree is reasonable. web-extension is currently the highest level package and will have highest level e2e tests. It's only purpose right now is to set up web extension messaging and it exports some cip30-specific setup code for that as well.

Consider this scenario: cip30 is extended for some other environment: (android, desktop wallet etc.), utilizing a different communication interface (like named pipes). You'd still be able to use most of the stuff that currently resides in cip30 package, but you wouldn't want to have a dependency on `web-extension`.

Reply:
OK, I'm with you on that.